HW1/P3/P3.py

```diff
@@ -0,0 +1,18 @@
+import findspark
+findspark.init()
+import pyspark
+sc = pyspark.SparkContext(appName='anagrams')
+
+with open('EOWL_words.txt', 'r') as rfile:
+	words = rfile.read().split('\n')
+
+lex_order = lambda s: ''.join(sorted(list(s)))
+
+words_rdd = sc.parallelize(words, 10)
+lex_to_word = words_rdd.map(lambda w: (lex_order(w), w))
+lex_to_wordlist = lex_to_word.groupByKey()#.mapValues(list)
```

Comment:
I don't use the whole group of words in the next line, and I don't think reducing by an identity fucntion would speed anything up. It might have been possible to reduce by length and then only fetch the contents of the largest group.